core/arch/x86/x86.asm

```diff
@@ -1791,11 +1791,11 @@ GLOBAL_LABEL(dr_setjmp:)
         /* PR 206278: for try/except we need to save the signal mask */
         mov      REG_XDX, ARG1
         push     REG_XDX /* preserve */
-# if defined(MACOS) && !defined(X64)
+# ifndef X64
```

Comment:
This and the #if below used to be Mac-only and unlike all the other changes in this PR here it's expanded to Windows too: is that intentional?  32-bit Windows has no stack alignment beyond 4 in its ABI.  I expected all the changes to be s/MACOS/UNIX/?

Reply:
this affects only UNIX, see `#ifdef UNIX` above

Reply:
OK.